README.md

```diff
@@ -84,6 +86,33 @@ All of the specified resources in the IAM policy do not have to exist prior to t
 }
 ```
 
+## Terraform Deployment
+
+Here is an example of how you would use the module in your own terraform deployment:
+
+```text
+module "github-actions-runner-aws" {
+  source = "github.com/cmsgov/github-actions-runner-aws?ref=v1.0.0"
+
+  # ECR variables
+  container_name          = "github-runner"
+  allowed_read_principals = local.principals_ro_access
+  ci_user_arn             = data.aws_iam_user.github_ci_user.arn
+
+  # ECS variables
+  environment               = "dev"
+  ecs_desired_count         = 0
```

Comment:
Maybe it makes sense to remove `ecs_desired_count` as a variable now that the count is being managed by the actions workflow

Reply:
I think it's still worth having the flexibility. it is optional, and with the updated docs, it's more clear it's optional/has a default of 0